### PR TITLE
added rough site framework to index.html

### PR DESCRIPTION
**Pull request by alexWaligorski**
dear fishies, please take a look! thx!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,71 @@
...
+            Digital Art and Creative Coding &#128073;
+            <a href="https://nodeforum.org/">NODE Forum</a>
+          </li>
+          <li>
+            Creative Coding Workshops for Kids &#128073;
+            <a href="https://www.digitale-welten.org/">Digitale Welten</a>
+          </li>
+          <li>
+            My dog &#128073;
+            <a href="https://i.redd.it/gan6b6zhn5f21.jpg">Woof!</a>
```
Comment (liebeckm) [inline index.html:50]: what a cutie! 😍

File: index.html [hunk 2]
```diff
@@ -4,7 +4,71 @@
...
+          probably this will change into a list tomorrow...because people use
+          linkedin and stuff...which must be linked...and has icons i should
+          use... goodnight!
+        </p>
+      </section>
+    </main>
+    <footer>
+      <nav>
+        <ul>
+          <li><a>Imprint</a></li>
```
Comment (liebeckm) [inline index.html:67]: href references for <a> tags in the footer are missing yet

**Review by liebeckm — COMMENTED** [review]
Guess it's fine 👍 some href's are still missing in footer but probably bc there is no site where they could link to.. yet :)

**Review by LuYujin9 — APPROVED** [review]
Du hast wirklich gut geschrieben!

File: index.html [hunk 3]
```diff
@@ -4,7 +4,71 @@
...
+        <a href="#contact">Contact</a>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h2 id="about-me">Alex, who?</h2>
+        <figure>
+          <img
+            src="https://nodeforum.org/wp-content/uploads/alexandra_waligorski_1-1.jpg"
+          />
```
Comment (PReichetanz) [inline index.html:24]: `` tags should always have an `alt` attribute stating what the image is showing. Only if the image is purely decorative, you can use `alt=""`.

**Pull request merged**